gulpfile.js/tasks/build.js

```diff
@@ -1,17 +1,20 @@
+const glob = require('tiny-glob');
+
 const { src, dest, parallel } = require('gulp');
 
 const sass = require('gulp-sass');
 const rename = require('gulp-rename');
 const postcss = require('gulp-postcss');
 const cssnano = require('cssnano');
+sass.compiler = require('sass');
```

Comment:
This can be a separate PR, but I would find it easier to manage things if the JavaScript and CSS build tasks were in their own files rather than in one big file.

Reply:
:+1: I can do that in a separate PR